README.md

```diff
@@ -61,7 +61,7 @@ You can change the HTML/CSS layout if you need it.
 ## Deploy and Pull Request
 
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_2048_game/)
+    - [DEMO LINK](https://Diana2369.github.io/js_2048_game/)
```

Comment:
Add this demo link to the PR description 
<img width="968" alt="image" src="https://github.com/user-attachments/assets/d1d8b511-a4e7-4af0-9d7c-22c94e96da4e" />